Display the device name

chardinson opened the pull request:
This code is a Python script that uses the Bleak module to connect to a Bluetooth LE device with the specified MAC address and read a specific character in a given service. Then, it prints the read value as a UTF-8 text string.

chardinson committed:
Display the device name

This code is a Python script that uses the Bleak module to connect to a Bluetooth LE device with the specified MAC address and read a specific character in a given service. Then, it prints the read value as a UTF-8 text string.

eni23 commented:
Hey,

Thanks a lot :)

I would actually write it something like this, since your one gives the following warning
```
xxx.py:12: DeprecationWarning: There is no current event loop
```
A event loop is not needed there and it is always a good idea to have `if __name__ == '__main__':` even when its overkill in this example. Also to keep it consistent with my scripts, i would suggest to add a shebang to the first line of the file (`#!/usr/bin/env python3`)


```python
#!/usr/bin/env python3

UE_BOOM_MAC = "10:94:97:05:5F:38"
service_uuid = "00001800-0000-1000-8000-00805f9b34fb"
characteristic_uuid = "00002a00-0000-1000-8000-00805f9b34fb"

async def main():
    async with BleakClient(UE_BOOM_MAC) as client:
        value = await client.read_gatt_char(characteristic_uuid)
        print(str(value, "utf-8"))

if __name__ == '__main__':
    asyncio.run(main())

```

chardinson committed:
Loop removed and better consistency with the other scripts

Removed the loop for errors and unnecessary use in the script.

Changes in the code that make it more consistent with other scripts.

chardinson commented:
> Hey,
> 
> Thanks a lot :)
> 
> I would actually write it something like this, since your one gives the following warning
> 
> ```
> xxx.py:12: DeprecationWarning: There is no current event loop
> ```
> 
> A event loop is not needed there and it is always a good idea to have `if __name__ == '__main__':` even when its overkill in this example. Also to keep it consistent with my scripts, i would suggest to add a shebang to the first line of the file (`#!/usr/bin/env python3`)
> 
> ```python
> #!/usr/bin/env python3
> 
> UE_BOOM_MAC = "10:94:97:05:5F:38"
> service_uuid = "00001800-0000-1000-8000-00805f9b34fb"
> characteristic_uuid = "00002a00-0000-1000-8000-00805f9b34fb"
> 
> async def main():
>     async with BleakClient(UE_BOOM_MAC) as client:
>         value = await client.read_gatt_char(characteristic_uuid)
>         print(str(value, "utf-8"))
> 
> if __name__ == '__main__':
>     asyncio.run(main())
> ```

Thank you for your comments and the changes in the code.

I have already applied the proposed changes.
